Framework/include/QualityControl/ServiceDiscovery.h

```diff
@@ -34,6 +34,7 @@ class ServiceDiscovery
  public:
   /// Sets up CURL and health check
   /// \param url 		Consul URL
+  /// \param name
```

Comment:
indeed, i forgot about it. Could you add the argument description?
```suggestion
  /// \param name		Service name
```